### DISCUSSION TITLE
Problem Load Data "<select>" @itwin/itwinui-react

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1379] [Q&A]
Hello good morning everyone,
I have a "<select" inside a Dialog that fails to load the data inside, vi that I've used it a lot, I can't figure out why it shouldn't load the data.

for the dialog and the select I use "@itwin/itwinui-react": "^1.48.2"

I attach the debug done on chrome which certifies the presence of the data.

Thanks for the precious help

<img width="633" alt="problem load Select" src="https://github.com/iTwin/iTwinUI/assets/57446652/8aeeb62c-7ce7-40cf-bb0c-431c1f823bfa">

**Comment by mayank99** [comment]
Hey @Paolo-lab. It's hard to help without more information about what you mean by "load the data". How are you passing the options? Are you able to reproduce this issue in an isolated [codesandbox](https://codesandbox.io/s/github/iTwin/iTwinUI/tree/main/cra-sandbox?file=/src/App.tsx)?

**Reply by r100-stack** [reply]
@Paolo-lab, just checking-in, were you able to solve the problem? If not, it would be great if you can share a [codesandbox](https://codesandbox.io/s/github/iTwin/iTwinUI/tree/main/cra-sandbox?file=/src/App.tsx) so we can try to reproduce and debug the problem.
